### DISCUSSION TITLE
Add support for Bunny.net Stream

### DISCUSSION THREAD

**Discussion by tony-rsrv** [mediacms-io/mediacms#670] [Feature Requests]
**Describe the feature you'd like**
Add support for bunny.net Stream. This service allows you to upload the videos and they will handle transcoding and delivery. The API is also pretty simple and they support webhooks to notify once the videos have transcoded. I think this is a great alternative to a complete S3 implementation.

**Comment by cirrusflyer** [comment]
This would be cool.
